moc_sprint_tools/cli.py

```diff
@@ -9,6 +9,7 @@
 from moc_sprint_tools import sort_cards_by_priority
 from moc_sprint_tools import cards_missing_from_backlog
 from moc_sprint_tools import close_sprint_boards
+from moc_sprint_tools import create_sprint_boards
```

Comment:
In addition to importing the module here, you need to  add the command at the end of the file:

```
main.add_command(create_sprint_boards.main)
```